template/src/pageComponents/mint/steps/MintProcessingStep/MintProcessingStep.tsx

```diff
@@ -0,0 +1,33 @@
+import { SymbolIcon } from '@radix-ui/react-icons';
+import clsx from 'clsx';
+
+export default function MintProcessingStep() {
```

Comment:
I see we create this Issue for later https://github.com/coinbase/build-onchain-apps/issues/328, which is great.

I was thinking if we should also find a way to consolidate or abstract how states work in a contract. So it's easier for a developer re-build them.